From: jg@suse.de
Subject: Set the default MAC address on prism54
Patch-Mainline: no

prism54 devices should set the default MAC address to
00:00:00:00:00:00, so that other tools have a chance to find out that
this is not the real MAC address.

Acked-by: Hannes Reinecke <hare@suse.de>

diff -urNp linux-2.6.4/drivers/net/wireless/prism54/islpci_dev.c linux-2.6.4.SUSE/drivers/net/wireless/prism54/islpci_dev.c
--- linux-2.6.4/drivers/net/wireless/prism54/islpci_dev.c	2004-03-29 13:58:00.000000000 +0200
+++ linux-2.6.4.SUSE/drivers/net/wireless/prism54/islpci_dev.c	2004-03-29 13:58:55.000000000 +0200
@@ -53,7 +53,9 @@
  * Of course, this is not the final/real MAC address. It doesn't
  * matter, as you are suppose to be able to change it anytime via
  * ndev->set_mac_address. Jean II */
-static const unsigned char	dummy_mac[6] = { 0x00, 0x30, 0xB4, 0x00, 0x00, 0x00 };
+/* Set dummy address to 00:00:00:00:00:00 so tools have a chance
+   to find out that this is no real mac address - jg */
+static const unsigned char	dummy_mac[6] = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
 
 /******************************************************************************
     Device Interrupt Handler
